content/intro-to-storybook/angular/en/test.md

```diff
@@ -4,19 +4,19 @@ tocTitle: 'Testing'
 description: 'Learn the ways to test UI components'
 ---
 
-No Storybook tutorial would be complete without testing. Testing is essential to creating high quality UIs. In modular systems, miniscule tweaks can result in major regressions. So far we encountered three types of tests:
+No Storybook tutorial would be complete without testing. Testing is essential to creating high-quality UIs. In modular systems, minuscule tweaks can result in major regressions. So far, we have encountered three types of tests:
 
 - **Manual tests** rely on developers to manually look at a component to verify it for correctness. They help us sanity check a component’s appearance as we build.
```

Comment:
Do we need to add Interaction testing here?

Reply:
@domyen I thought about it, but to avoid confusing readers as technically we're not adding interaction testing but only letting them know that they can interact with components using the play function. Once 6.5 is out yes we'll need to adjust and incorporate it